### PR TITLE
fix: missing/false port exposures

### PR DESCRIPTION
**Pull request by kilrau**
Closes https://github.com/ExchangeUnion/xud-docker/issues/373

### CODE REVIEW

File: images/utils/launcher/config/mainnet.conf [hunk 1]
```diff
@@ -108,4 +108,5 @@
 # 8885 - P2P port
 # 8886 - gRPC port
 # 8887 - raiden resolver port
-#expose-ports = ["8885", "8886", "8887"]
+# 8080 - webproxy port
+#expose-ports = [8885,8886,8887,8080]
```
Comment (michael1011) [inline images/utils/launcher/config/mainnet.conf]: Missing blank line at the end of file

File: images/utils/launcher/config/testnet.conf [hunk 2]
```diff
@@ -90,22 +90,23 @@
...
+# 20010 - REST port
+#expose-ports = ["19735:9735", "20009:10009", "20010:10010"]
 
 [lndltc]
 # 20735 - P2P port
-# 10009 - gRPC port
-# 10010 - REST port
-#expose-ports = ["20735", "20009:10009", "20010:10010"]
+# 21009 - gRPC port
+# 21010 - REST port
```
Comment (michael1011) [inline images/utils/launcher/config/testnet.conf]:
```suggestion
# 21010 - REST port
```

File: images/utils/launcher/config/mainnet.conf (hunk 1 above)
Comment (reliveyy) [inline images/utils/launcher/config/mainnet.conf]: Missing newline in the end of file.

File: images/utils/launcher/config/simnet.conf [hunk 3]
```diff
@@ -11,29 +11,25 @@
...
-# 18556 - JSON-RPC port
-#expose-ports = ["18556"]
-
 [lndbtc]
 # 29735 - P2P port
-# 10009 - gRPC port
-# 10010 - REST port
-#expose-ports = ["29735", "10009", "10010"]
+# 30009 - gRPC port
+# 30010 - REST port
```
Comment (reliveyy) [inline images/utils/launcher/config/simnet.conf]: Extra space in the end of line.

File: images/utils/launcher/config/simnet.conf [hunk 4]
```diff
@@ -11,29 +11,25 @@
...
+# 30010 - REST port 
+#expose-ports = ["29735:9735", "30009:10009", "30010:10010"]
 
 [lndltc]
 # 30735 - P2P port
-# 10009 - gRPC port
-# 10010 - REST port
-#expose-ports = ["30735", "20009:10009", "20010:10010"]
+# 31009 - gRPC port
+# 31010 - REST port
```
Comment (reliveyy) [inline images/utils/launcher/config/simnet.conf]: Extra space in the end of line.

File: images/utils/launcher/config/testnet.conf (hunk 2 above)
Comment (reliveyy) [inline images/utils/launcher/config/testnet.conf]: Extra space in the end of line.

File: images/utils/launcher/config/simnet.conf (hunk 3 above)
Comment (michael1011) [inline images/utils/launcher/config/simnet.conf]:
```suggestion
# 30010 - REST port
```

**Pull request merged**